code/helpers/overlays.dm

```diff
@@ -0,0 +1,13 @@
+/atom/proc/create_overlay(obj/item/equippable/from)
```

Comment:
`/proc/create_overlay(atom/from)`
or perhaps `create_image` would be more accurate.